frontend/nuxt.config.js

```diff
@@ -3,7 +3,7 @@ export default {
    ** Nuxt rendering mode
    ** See https://nuxtjs.org/api/configuration-mode
    */
-  mode: 'universal',
+  ssr: true,
```

Comment:
I think we should make the site fully static, for better performance and easier deployments. Can you put `target: 'static'` here instead?

Reply:
Right, I agree

Reply:
My bad, it's actually the way the data is fetched that breaks SSG. I'll fix this myself.

But I still think the `ssr: true` might conflict with SSG. Can you remove this sections altogether (delete lines 2 to 6) ?

Reply:
Done